### ISSUE TITLE
Bug in the last item animation

### ISSUE THREAD

**Issue by Iuriy-Budnikov** [zerodevx/svelte-toast#61]
![ScreenFlow](https://user-images.githubusercontent.com/3968373/202481591-a9c7db9e-c92b-42c0-a74a-5868959c8863.gif)

**Comment by zerodevx** [comment]
Yes, I noticed this too but I'm not sure if it's fixable. This happens when 2 or more toasts are removed at about the same time. `svelte-toast` actually delegates all animation work to be handled by Svelte, so it feels like it's a Svelte issue.

**Comment by Iuriy-Budnikov** [comment]
> Yes, I noticed this too but I'm not sure if it's fixable. This happens when 2 or more toasts are removed at about the same time. `svelte-toast` actually delegates all animation work to be handled by Svelte, so it feels like it's a Svelte issue.

Does it make sense to add some delay?

**Comment by zerodevx** [comment]
Normally I'd rather let Svelte do the heavy lifting re animations and batching - but I guess it's possible to place `pop` requests into a rate-limiting queue. I'll test this out once I have some time.

**Comment by m-gschwandtner** [comment]
Setting "toastContainerLeft" to a fixed value seems to fix the position,
items removed at the same time are still on top of one another.

I just used this: `--toastContainerLeft: calc(100vw - 350px);`

A bit stuck on the overlapping issue, tough.

![Peek 2023-01-06 06-39](https://user-images.githubusercontent.com/84477901/210938130-c7fec4ef-7784-4bdf-9893-4329a196450b.gif)
